xml/installation-installation-configure_lbaas.xml

```diff
@@ -161,7 +161,7 @@
    <step>
     <para>
      The full path and name for the Amphora image is
-     <filename>/srv/www/suse-12.3/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-1.21.noarch.rpm</filename>
+     <filename>/srv/tftpboot/suse-12.4/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-13.157.noarch.rpm</filename>
```

Comment:
Are you sure this is correct. On my patched SOC 9 it is in /srv/www/suse-12.4/x86_64/repos/Cloud/suse/noarch

Reply:
also is that version correct? This seems to imply a different version string .

https://build.suse.de/package/live_build_log/Devel:Cloud:9/openstack-octavia-amphora-image/SLE_12_SP4/x86_64

Reply:
I got the path from @nicolasbock  , we should figure out why his environment has a different path than you're seeing

Reply:
On my fresh `develcloud9` env the path is
```
/srv/tftpboot/suse-12.4/x86_64/repos/Cloud/suse/noarch/openstack-octavia-amphora-image-x86_64-0.1.0-13.157.noarch.rpm
```

Reply:
I don't see much in `/srv/www`. Just two directories but no files.

Reply:
is that CLM or Crowbar?

Reply:
Crowbar

Reply:
My understanding from @KeithMnemonic is that the CLM path differs from the Crowbar path, will need him to confirm (or someone else with an Octavia CLM system setup)